python/ManagingMultipleLargeScaleJobs/ManagingMultipleLargeScaleJobs.ipynb

```diff
@@ -0,0 +1,443 @@
+    "    s2_cube = s2_cube.apply_dimension(\n",
+    "        dimension = \"t\",\n",
+    "        process = \"array_interpolate_linear\"\n",
+    "    )\n",
+    "\n",
+    "    s2_cube.save_result(format = \"GTiff\")\n",
+    "    \n",
+    "    return s2_cube.create_job(\n",
+    "        title='MultiBackendJobManager - Example'\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ef49e8a4",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## Managing Jobs Using MultiBackendJobManager\n",
+    "\n",
+    "With our jobs database and job definition set up, we can now run the jobs using the `MultiBackendJobManager`. This involves defining a path to where we will store the job tracker which contains the job statuses and metadata.\n",
+    "\n",
+    "### Steps to Run the Jobs:\n",
+    "\n",
+    "1. Provide a path towards the job tracker:\n",
+    "   The predefined jobs database contains all the input parameters for the jobs we intend to run with the MultiBackendJobManager. Typically, this database will serve as the foundation for the job tracker. By specifying a unique path, the job tracker will be created, incorporating all the information from the original jobs database. Additionally, this file will be continuously updated to reflect actual job statuses and usage credits.\n",
+    "\n",
+    "   The job tracker will also include a 'status' column, allowing you to easily monitor which jobs have been created, are currently running, have finished, or encountered an error.\n",
+    "\n",
+    "   **Caution:**\n",
+    "   If the specified path points to an existing jobs database (either a .csv or .parquet file), that file will be used as the job tracker, and the input jobs database will be disregarded.\n",
+    "   \n",
+    "<br>\n",
+    "\n",
+    "\n",
+    "2. Initialize the MultiBackendJobManager:\n",
+    "   We create an instance of the `MultiBackendJobManager` and add a backend of our choice, which will be responsible for executing the jobs.\n",
+    "   \n",
+    "   **Caution:**\n",
+    "   The number of parallel jobs refers to how many jobs the job manager tracks at once. However, the backend itself limits the actual number of jobs that can start and run at the same time to 2.\n",
+    "\n",
+    "<br>\n",
+    "\n",
+    "3. Run Multiple Jobs:\n",
+    "   Use `manager.run_jobs` to create the desired jobs and send them to the backend.\n",
+    "   The selected job tracker will be updated on the actual job statuses and usage metris."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "52f22b3e",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Authenticated using refresh token.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Generate a unique name for the tracker\n",
+    "job_tracker = datetime.datetime.now().strftime(\"%Y_%m_%d_%H_%M_%S\") + '_job_tracker.parquet'\n",
```

Comment:
Now that I think about it, giving a timestamp to the job_tracker is maybe not best practice? The whole point of the job manager is to be able to resume your jobs from the job_tracker. If you give a timestamp to the job_tracker, it will create a new one each time the script is run.